robot.py

```diff
@@ -131,14 +136,20 @@ def createObjects(self):
 
         self.spin_rate = 2.5
 
+    def autonomous(self):
+        self.timer = 135
```

Comment:
If this is intended to be a match timer, the DS already sends an unofficial match time to the robot.